Day4_excercise.rb

```diff
@@ -0,0 +1,68 @@
+print "Please create your password: "
+password = gets.chomp
+numbers = ["0", "1", "2", "3", "4", "5", "6", "7", "8", "9"]
+symbols = ["@", "%", "*", "!"]
+uppercharacters = ("A".."Z").to_a
+lowercharacters = ("a".."z").to_a
+hasNumber = false
+hasSymbol = false
+hasUpper = false
+hasLower = false
+hasMoreThan8Characters = false
+password.each_char do |element| #switch string to chars
+  numbers.each do |number|
```

Comment:
These checks all look similar except the array you use is different. Maybe you can make it a method.